samples/subsys/ipc/openamp_rsc_table/src/main_remote.c

```diff
@@ -31,9 +30,16 @@ LOG_MODULE_REGISTER(openamp_rsc_table, LOG_LEVEL_DBG);
 #define SHM_START_ADDR	DT_REG_ADDR(SHM_NODE)
 #define SHM_SIZE		DT_REG_SIZE(SHM_NODE)
 
+#define MAX_TTY_EPT  1
```

Comment:
This doesn't appear to be used?

Reply:
well spotted! residue of a cleaning,  to suppress